ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/TerminateQueryEntity.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TerminateQueryEntity extends KsqlEntity {
+
+  private final String queryId;
+  private final boolean wasTerminatedLocally;
+
+  @JsonCreator
+  public TerminateQueryEntity(
+      @JsonProperty("statementText") final String statementText,
+      @JsonProperty("queryId") final String queryId,
+      @JsonProperty("wasTerminatedLocally") final boolean wasTerminatedLocally
```

Comment:
This is also used to pass on the aggregated response in which case it really means wasTerminatedRemotely.  Maybe to make it more consistent, we can just call it wasTerminated?